### ISSUE TITLE
Will this be merged upstream?

### ISSUE THREAD

**Issue by Max-Sum** [extfuse/extfuse#2]
This project will benefit a lot for [trapexit/mergerfs](https://github.com/trapexit/mergerfs). I wonder if the changes will made it to upstream linux.

**Comment by ashishbijlani** [comment]
Yes, I'm working on the patch. I hope to send the first one out early next month.

**Comment by HZ89** [comment]
is any progress on the patch?
